### PR TITLE
Added a technical introduction to sso

### PR DESCRIPTION
**Pull request by vdlmark**

### CODE REVIEW

File: data/blog/a-technical-introduction-to-sso.md [hunk 1]
```diff
@@ -0,0 +1,97 @@
...
+The term Single Sign-On has multiple meanings in the industry. A well-known one is to describe users signing into a platform once and gaining access to multiple applications. However, an SSO solution can provide a lot more than just simplified authentication. It can provide:
+
+- Authorization based on roles to restrict or allow access to certain resources.
+- Centralized user management, making it easier to audit and be compliant with certain certifications.
+- Supporting multiple authentication protocols to connect to other identity providers.
+- Connect legacy systems to integrate with existing user bases.
+
+In addition, these solutions provide the benefit of having the authentication mechanism centralized and based on standards. This means not having to deal with authentication, or storing of user data in applications that need some form of login functionality.
+
+In this first part of the SSO blog series, we will answer the following questions: 'What does an SSO solution provide other than simplify logins for your users?', 'What protocols are relevant in today's SSO solutions?' and 'What authentication flows are available and what makes them secure?'
```
Comment (DaveBitter) [inline data/blog/a-technical-introduction-to-sso.md]: Nice! I've merged functionality to build series on the tech_hub a few minutes ago. Let's have a short call to turn this into an actual series

File: data/blog/a-technical-introduction-to-sso.md [hunk 2]
```diff
@@ -0,0 +1,97 @@
+---
+title: 'A technical introduction to SSO'
+date: '2022-08-16'
+tags: ['security']
+summary: 'Single Sing-on has been around for years but on what standards is it working?'
+authors: ['daan-beulen', 'mark-van-der-linden']
+theme: 'blue'
+---
```
Comment (DaveBitter) [inline data/blog/a-technical-introduction-to-sso.md]: Could you add a "teaser image"?

Based on the comment below about the series, I could also get you in touch to have artwork for the entire series designed

File: data/blog/a-technical-introduction-to-sso.md [hunk 3]
```diff
@@ -0,0 +1,97 @@
...
+title: 'A technical introduction to SSO'
+date: '2022-08-16'
+tags: ['security']
+summary: 'Single Sing-on has been around for years but on what standards is it working?'
+authors: ['daan-beulen', 'mark-van-der-linden']
```
Comment (DaveBitter) [inline data/blog/a-technical-introduction-to-sso.md]: I just noticed that on the article page it shows both authors, but on the overview page it doesn't. Perhaps follow the same design @sanderdejong88 ?

**Comment by DaveBitter** [comment]
Lekker bezig! Excited about this series!

File: data/blog/a-technical-introduction-to-sso.md [hunk 4]
```diff
@@ -0,0 +1,97 @@
+---
+title: 'A technical introduction to SSO'
+date: '2022-08-16'
+tags: ['security']
+summary: 'Single Sing-on has been around for years but on what standards is it working?'
```
Comment (DaveBitter) [inline data/blog/a-technical-introduction-to-sso.md]: Typo ;P

File: data/blog/a-technical-introduction-to-sso.md [hunk 5]
```diff
@@ -0,0 +1,97 @@
...
+
+In addition, these solutions provide the benefit of having the authentication mechanism centralized and based on standards. This means not having to deal with authentication, or storing of user data in applications that need some form of login functionality.
+
+In this first part of the SSO blog series, we will answer the following questions: 'What does an SSO solution provide other than simplify logins for your users?', 'What protocols are relevant in today's SSO solutions?' and 'What authentication flows are available and what makes them secure?'
+
+## SSO Flow
+
+Before we go into the details of what SSO solutions provide, the image below will be explained to give a better understanding of the standard Single Sign-On use case.
+
+![SSO flow](/articles/a-technical-introduction-to-sso/sso-flow.png)
```
Comment (mrtnvh) [inline data/blog/a-technical-introduction-to-sso.md]: Can we replace this with more generic logo's?

File: data/blog/a-technical-introduction-to-sso.md (hunk 1 above)
Comment (DaveBitter) [reply data/blog/a-technical-introduction-to-sso.md]: Just spoke to @vanhoofmaarten  this is a ported article and turned out to be a standalone article. Series functionality not needed.

File: data/blog/a-technical-introduction-to-sso.md (hunk 2 above)
Comment (DaveBitter) [reply data/blog/a-technical-introduction-to-sso.md]: Just spoke to @vanhoofmaarten  this is a ported article and turned out to be a standalone article. Series artwork not needed. A standalone image should be fine

File: data/blog/a-technical-introduction-to-sso.md (hunk 1 above)
Comment (DaveBitter) [inline data/blog/a-technical-introduction-to-sso.md]: As it is not a series, could you change this?

File: data/blog/a-technical-introduction-to-sso.md [hunk 6]
```diff
@@ -0,0 +1,97 @@
...
+
+### Implicit flow
+
+The implicit flow is not a recommended flow. When this flow was created, browsers had limitations and several workarounds had to be used. The flow has many steps in common with the authorization code flow but misses some key aspects of making it secure. The main difference is that the access token is sent back to the application directly in the redirection step. When an access token is sent back in a redirection, there are multiple ways for attackers to intercept this token.
+
+![Implicit flow](/articles/a-technical-introduction-to-sso/implicit-flow.png)
+
+## Summary
+
+In this first post we've gone over what we see as Single Sign-On and the different protocols that are relevant right now. We've also gone more in-depth on the flows of the OAuth2.0 and OIDC protocols. In the next post, we will look at four industry leaders in the SSO space and their product offerings. We will make a comparison of their features and will offer you criteria that are important to look at when making your decision for the best solution.
```
Comment (DaveBitter) [inline data/blog/a-technical-introduction-to-sso.md]: As it is a single post, could you rewrite this?

File: data/blog/a-technical-introduction-to-sso.md [hunk 7]
```diff
@@ -0,0 +1,97 @@
+---
+title: 'A technical introduction to SSO'
+date: '2022-08-16'
```
Comment (DaveBitter) [inline data/blog/a-technical-introduction-to-sso.md]: This is a ported article, could you add the original date of publishing?

File: data/blog/a-technical-introduction-to-sso.md (hunk 3 above)
Comment (sanderdejong88) [reply data/blog/a-technical-introduction-to-sso.md]: https://github.com/iodigital-com/io-technology/pull/69 yes, being worked on.

File: data/blog/a-technical-introduction-to-sso.md (hunk 1 above)
Comment (vdlmark) [reply data/blog/a-technical-introduction-to-sso.md]: Changed it to a series.

File: data/blog/a-technical-introduction-to-sso.md (hunk 6 above)
Comment (vdlmark) [reply data/blog/a-technical-introduction-to-sso.md]: Changed it to a series.

File: data/blog/a-technical-introduction-to-sso.md (hunk 7 above)
Comment (lucienimmink) [reply data/blog/a-technical-introduction-to-sso.md]: a-technical-introduction-to-sso: 22-09-2020
an-overview-of-sso-products: 15-10-2020
sso-in-action: 26-02-2021

**Pull request merged**